### PR TITLE
[system] Add Windows Firewall events

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Add Windows Firewall connection and packet drop events.
- Add tests for event codes: 5152, 5156, 5157, 5158
- Add source ECS fields
- Add destination ECS fields
- Add network ECS fields

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #5254

### CODE REVIEW

File: packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml [hunk 1]
```diff
@@ -2185,6 +2215,18 @@ processors:
             ctx.winlog.event_data.put("AccessListDescription", results);
           }
         }
+        if (ctx?.winlog?.event_data?.Direction != null) {
```
Comment (efd6) [inline packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml:2218]: `s/ctx\?\./ctx./g` (`ctx` is guaranteed to be non-null).

File: packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml [hunk 2]
```diff
@@ -2890,6 +2932,100 @@ processors:
...
+      ignore_failure: false
+      tag: Add Connection Events
+      description: Add Connection Events
+      source: |-
+        if (ctx?.event?.code == null ||
+            !["5152", "5156", "5157", "5158"].contains(ctx.event.code)) {
+          return;
+        }
+        
+        //DestAddress to destination.ip and related.ip
```
Comment (efd6) [inline packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml:2946]:
```suggestion
        // DestAddress to destination.ip and related.ip
```
(for readability — also below)

**Review by efd6 — COMMENTED** [review]
What is the source of the events?

**Comment by LaZyDK** [comment]
> What is the source of the events?

Events from the system.security dataset from one of our clusters where these audits have been enabled by GPO.

**Comment by efd6** [comment]
Thanks. We'll note that as "User-provided test cases" in the commit message.

File: packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml (hunk 1 above)
Comment (LaZyDK) [reply packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml]: Done :)

**Comment by efd6** [comment]
It looks like it's not properly configured yet, so no.

**Comment by LaZyDK** [comment]
Resolved conflicts. Ready to test.

**Comment by LaZyDK** [comment]
I am ready to fix conflicts if #6528 is pulled before this one :)

**Comment by LaZyDK** [comment]
> Details

Agent error. Haven't seen that before.

**Comment by LaZyDK** [comment]
What is broken?

**Comment by LaZyDK** [comment]
Any updates?

**Comment by LaZyDK** [comment]
Ready for test

**Comment by LaZyDK** [comment]
@efd6 Resolved conflicts again. 
Can we merge this before the next conflicts or is anything in the way of that?

**Comment by efd6** [comment]
@LaZyDK I'm afraid it's not under my control. We don't own this package.

**Comment by jamiehynds** [comment]
@SubhrataK as the codeowners, could someone from your team please review/merge this PR. It's adding Windows Firewall Events to the Windows Security Events datastream in the System package.

**Review by efd6 — APPROVED** [review]

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Comment by LaZyDK** [comment]
Resolved conflicts and ran tests again. Ready to merge after CI tests.

**Comment by LaZyDK** [comment]
@SubhrataK or @ishleenk17 I resolved conflicts again. Waiting for a merge.

**Comment by ishleenk17** [comment]
> @SubhrataK or @ishleenk17 I resolved conflicts again. Waiting for a merge.

Will merge it once the CI passes. Thanks

**Comment by nicpenning** [comment]
I am of the opinion this needs to go to the Windows integration.  What are your guys thoughts?

My plan was to introduce this data stream after we can get through AppLocker, but @LaZyDK beat me to it 😃

**Comment by nicpenning** [comment]
Also, thanks for fixing that improper PR link, you da man!

**Pull request merged**